src/main/resources/log4j2.xml

```diff
@@ -8,10 +8,19 @@
         <Console name="Console" target="SYSTEM_OUT">
             <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n" charset="UTF-8"/>
         </Console>
+        <RollingRandomAccessFile name="RollingRandomAccessFile" filename="logs/futrzak.log" filePattern="logs/$${date:yyyy-MM}/futrzak-%d{MM-dd-yyyy}-%i.log.gz">
+            <PatternLayout pattern="%d{HH:mm:ss.SSS} [%t] %-5level %logger{36} - %msg%n" charset="UTF-8"/>
+            <DefaultRolloverStrategy max="10"/>
+            <Policies>
+                <TimeBasedTriggeringPolicy interval="1"/>
+                <SizeBasedTriggeringPolicy size="250 MB"/>
+            </Policies>
+        </RollingRandomAccessFile>
```

Comment:
daje approva choć nawet nie wiem co to jest (jakiś plik xml?)

Reply:
To po to żeby komunikaty(logi) w konsoli zapisywały się do pliku w celu historii.